yaspin/yaspin.py

```diff
@@ -183,14 +183,32 @@ def show(self):
             sys.stdout.write("\r")
             self._clear_line()
 
-    def write(self, text):
-        """Write text in the terminal without breaking the spinner."""
+    def write(self, text, lstrip=True, rstrip=True):
```

Comment:
After some rethinking I'm coming to conclusion that `write(self, text, strip=True)` would be a better API. `strip=True` will allow to save a default `write` behavior and `strip=False` allows user to strip a text in desired way before calling the `write` method. I think such approach would be more simple and flexible.

What do you think?

Reply:
Yep, true.